guides/runbooks/orca-quality-of-service.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: single
+title:  "Orca Quality of Service"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+**EXPERIMENTAL**: This feature is still in an early adoption / experimental phase. 
+While you can use it today (Orca v6.71.0), Netflix is still currently running this in learning mode / judiciously enabling in response to on-call events.
+
+Spinnaker ships with an optional Quality of Service (QoS) module that can be used to manage the amount of active executions running at any given time.
+By default, this QoS module is disabled, but can be enabled and tuned with a handful of knobs and different strategies.
+Before we dive into the configuration settings, we'll go over how QoS works.
+
+# QoS Internals
```

Comment:
Maybe change the heading from Internals to `How does QoS work?` This seems needed for configuration later but the heading Internals make me want to skip this section.